.gitignore

```diff
@@ -9,3 +9,4 @@ vendor
 nbproject
 phpcs-checkstyle.tmp
 composer.phar
+.vscode/
```

Comment:
Why is this change required? Environment specific files/directories should be added to your local ignore list: https://help.github.com/articles/ignoring-files/#create-a-local-gitignore